tsdoc-config/src/TSDocConfigFile.ts

```diff
@@ -217,6 +229,27 @@ export class TSDocConfigFile {
     this._tagDefinitions.push(tagDefinition);
   }
 
+  /**
+   * Adds a new item to the `supportedHtmlElements` array.
+   */
+  public addSupportedHtmlElement(htmlElement: string): void {
+    if (!this._supportedHtmlElements) {
+      this._supportedHtmlElements = new Set();
+    }
+    this._supportedHtmlElements.add(htmlElement);
+  }
+
+  /**
+   * Removes the explicit list of allowed html elements.
+   */
+  public clearSupportedHtmlElements(): void {
+    this._supportedHtmlElements = undefined;
+  }
+
+  public setReportUnsupportedHtmlElements(reportUnsupportedHtmlElements: boolean): void {
```

Comment:
Since this is a simple `boolean` with no validation constraints, it seems fine to expose this as a simple property like `noStandardTags` above.  That would also enable the caller to reset the state to `undefined` if desired.